securedrop/models.py

```diff
@@ -20,7 +20,16 @@
 from passphrases import PassphraseGenerator
 from sqlalchemy import Boolean, Column, DateTime, ForeignKey, Integer, LargeBinary, String, Text
 from sqlalchemy.exc import IntegrityError
-from sqlalchemy.orm import Query, backref, relationship
+from sqlalchemy.orm import (
+    Query,
+    RelationshipProperty,
+)
+from sqlalchemy.orm import (
+    backref as _backref,
+)
+from sqlalchemy.orm import (
+    relationship as _relationship,
+)
```

Comment:
You'd think this would be:


```suggestion
from sqlalchemy.orm import (
    Query,
    RelationshipProperty,
    backref as _backref,
    relationship as _relationship,
)
```

But Ruff really wants this way.